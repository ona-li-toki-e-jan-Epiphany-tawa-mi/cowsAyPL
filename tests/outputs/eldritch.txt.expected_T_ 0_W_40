/¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯\
| We are become one. F |
| lesh in the flesh.   |
\______________________/
                    \                  
                     \                 
                       ^__^            
                       (oo)\_______    
                       (__)\       )\/\
                         0 ||----w |   
                           ||     ||   
