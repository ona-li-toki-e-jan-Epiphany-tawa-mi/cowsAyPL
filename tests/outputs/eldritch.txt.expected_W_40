/¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯\
| We are become one. Flesh in th |
| e flesh.                       |
\________________________________/
                              \                  
                               \                 
                                 ^__^            
                                 (oo)\_______    
                                 (__)\       )\/\
                                     ||----w |   
                                     ||     ||   
