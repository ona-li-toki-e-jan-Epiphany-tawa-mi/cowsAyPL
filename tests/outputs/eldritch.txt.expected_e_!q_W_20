/¯¯¯¯¯¯¯¯¯¯¯¯\
| We are bec |
| ome one. F |
| lesh in th |
| e flesh.   |
\____________/
          \                  
           \                 
             ^__^            
             (!q)\_______    
             (__)\       )\/\
                 ||----w |   
                 ||     ||   
