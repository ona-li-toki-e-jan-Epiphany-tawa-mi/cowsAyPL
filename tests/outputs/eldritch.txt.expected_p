/¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯\
| + p We are become one. Flesh in the fles |
| h.                                       |
\__________________________________________/
                                        \                  
                                         \                 
                                           ^__^            
                                           (oo)\_______    
                                           (__)\       )\/\
                                               ||----w |   
                                               ||     ||   
