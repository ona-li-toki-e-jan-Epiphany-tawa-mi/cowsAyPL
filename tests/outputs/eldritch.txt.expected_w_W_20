/¯¯¯¯¯¯¯¯¯¯¯¯\
| We are bec |
| ome one. F |
| lesh in th |
| e flesh.   |
\____________/
          \                  
           \                 
             ^__^            
             (OO)\_______    
             (__)\       )\/\
                 ||----w |   
                 ||     ||   
