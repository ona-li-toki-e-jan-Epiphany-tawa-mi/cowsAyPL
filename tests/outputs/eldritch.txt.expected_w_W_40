/¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯\
| We are become one. Flesh in th |
| e flesh.                       |
\________________________________/
                              \                  
                               \                 
                                 ^__^            
                                 (OO)\_______    
                                 (__)\       )\/\
                                     ||----w |   
                                     ||     ||   
