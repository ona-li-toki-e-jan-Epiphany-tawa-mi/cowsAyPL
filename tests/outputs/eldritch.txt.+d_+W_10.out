/¯¯¯¯¯¯¯¯¯¯¯¯\
| We are bec |
| ome one. F |
| lesh in th |
| e flesh.   |
\____________/
        \
         \
           ^__^
           (XX)\_______
           (__)\       )\/\
            U  ||----w |
               ||     ||
